gcc 2.8.1 build sometimes fails

The build is done with `-j` which allows it to succeed _most_ of the time. Sometimes it'll fail because:

```
3.241 profile.c:48:10: fatal error: insn-config.h: No such file or directory
3.241    48 | #include "insn-config.h"
3.241       |          ^~~~~~~~~~~~~~~
```

**Edit**: gcc 2.5.7-psx and gcc 2.6.3-psx also sometimes fail with the same error about `insn-config.h`

--- a/gcc-2.5.7-psx.Dockerfile
+++ b/gcc-2.5.7-psx.Dockerfile
@@ -20,6 +20,7 @@ RUN patch -u -p1 gcc.c -i ../patches/gcc-2.5.7.c.patch
 RUN patch -u -p1 g++.c -i ../patches/g++-2.5.7.c.patch
 RUN patch -u -p1 config/mips/mips.h -i ../patches/mips-2.5.7.h.patch
 RUN patch -su -p1 < ../patches/psx-2.5.7.patch
+RUN touch insn-config.h
 
 RUN ./configure \
     --target=mips-sony-psx \
@@ -29,7 +30,7 @@ RUN ./configure \
     --host=i386-pc-linux \
     --build=i386-pc-linux
 
-RUN make -j cpp cc1 xgcc cc1plus g++ CFLAGS="-std=gnu89 -m32 -static -Dbsd4_4 -Dmips -march=i686 -DHAVE_STRERROR"
+RUN make --jobs $(nproc) cpp cc1 xgcc cc1plus g++ CFLAGS="-std=gnu89 -m32 -static -Dbsd4_4 -Dmips -march=i686 -DHAVE_STRERROR"
 
 COPY tests /work/tests
 RUN ./cc1 -quiet -O2 /work/tests/little_endian.c && grep -E 'lbu\s\$2,0\(\$4\)' /work/tests/little_endian.s

--- a/gcc-2.5.7.Dockerfile
+++ b/gcc-2.5.7.Dockerfile
@@ -28,7 +28,7 @@ RUN ./configure \
     --host=i386-pc-linux \
     --build=i386-pc-linux
 
-RUN make cpp cc1 xgcc cc1plus g++ CFLAGS="-std=gnu89 -m32 -static -Dbsd4_4 -Dmips -DHAVE_STRERROR"
+RUN make --jobs $(nproc) cpp cc1 xgcc cc1plus g++ CFLAGS="-std=gnu89 -m32 -static -Dbsd4_4 -Dmips -DHAVE_STRERROR"
 
 COPY tests /work/tests
 RUN ./cc1 -quiet -O2 /work/tests/little_endian.c && grep -E 'lbu\s\$2,0\(\$4\)' /work/tests/little_endian.s

--- a/gcc-2.6.0.Dockerfile
+++ b/gcc-2.6.0.Dockerfile
@@ -27,7 +27,7 @@ RUN ./configure \
     --host=i386-pc-linux \
     --build=i386-pc-linux
 
-RUN make cpp cc1 xgcc cc1plus g++ CFLAGS="-std=gnu89 -m32 -static -Dbsd4_4 -Dmips -DHAVE_STRERROR"
+RUN make --jobs $(nproc) cpp cc1 xgcc cc1plus g++ CFLAGS="-std=gnu89 -m32 -static -Dbsd4_4 -Dmips -DHAVE_STRERROR"
 
 COPY tests /work/tests
 RUN ./cc1 -quiet -O2 /work/tests/little_endian.c && grep -E 'lbu\s\$2,0\(\$4\)' /work/tests/little_endian.s

--- a/gcc-2.6.3-psx.Dockerfile
+++ b/gcc-2.6.3-psx.Dockerfile
@@ -17,6 +17,7 @@ RUN patch -u -p1 cp/g++.c -i ../patches/g++-2.6.3.c.patch
 RUN patch -su -p1 < ../patches/psx.patch
 
 RUN touch -c cp/parse.y cp/parse.h cp/parse.c
+RUN touch insn-config.h
 
 RUN ./configure \
     --target=mips-sony-psx \
@@ -26,7 +27,7 @@ RUN ./configure \
     --host=i386-pc-linux \
     --build=i386-pc-linux
 
-RUN make -j cpp cc1 xgcc cc1plus g++ CFLAGS="-std=gnu89 -m32 -static -Dbsd4_4 -Dmips -march=i686 -DHAVE_STRERROR"
+RUN make --jobs $(nproc) cpp cc1 xgcc cc1plus g++ CFLAGS="-std=gnu89 -m32 -static -Dbsd4_4 -Dmips -march=i686 -DHAVE_STRERROR"
 
 COPY tests /work/tests
 RUN ./cc1 -quiet -O2 /work/tests/little_endian.c && grep -E 'lbu\s\$2,0\(\$4\)' /work/tests/little_endian.s

--- a/gcc-2.6.3.Dockerfile
+++ b/gcc-2.6.3.Dockerfile
@@ -27,7 +27,7 @@ RUN ./configure \
     --build=i386-pc-linux
 
 
-RUN make -j cpp cc1 xgcc cc1plus g++ CFLAGS="-std=gnu89 -m32 -static -Dbsd4_4 -Dmips -DHAVE_STRERROR"
+RUN make --jobs $(nproc) cpp cc1 xgcc cc1plus g++ CFLAGS="-std=gnu89 -m32 -static -Dbsd4_4 -Dmips -DHAVE_STRERROR"
 
 COPY tests /work/tests
 RUN ./cc1 -quiet -O2 /work/tests/little_endian.c && grep -E 'lbu\s\$2,0\(\$4\)' /work/tests/little_endian.s

--- a/gcc-2.7.0.Dockerfile
+++ b/gcc-2.7.0.Dockerfile
@@ -27,7 +27,7 @@ RUN ./configure \
     --host=i386-pc-linux \
     --build=i386-pc-linux
 
-RUN make -j cpp cc1 xgcc cc1plus g++ CFLAGS="-std=gnu89 -m32 -static"
+RUN make --jobs $(nproc) cpp cc1 xgcc cc1plus g++ CFLAGS="-std=gnu89 -m32 -static"
 
 COPY tests /work/tests
 RUN ./cc1 -quiet -O2 /work/tests/little_endian.c && grep -E 'lbu\s\$2,0\(\$4\)' /work/tests/little_endian.s

--- a/gcc-2.7.1.Dockerfile
+++ b/gcc-2.7.1.Dockerfile
@@ -27,7 +27,7 @@ RUN ./configure \
     --host=i386-pc-linux \
     --build=i386-pc-linux
 
-RUN make -j cpp cc1 xgcc cc1plus g++ CFLAGS="-std=gnu89 -m32 -static"
+RUN make --jobs $(nproc) cpp cc1 xgcc cc1plus g++ CFLAGS="-std=gnu89 -m32 -static"
 
 COPY tests /work/tests
 RUN ./cc1 -quiet -O2 /work/tests/little_endian.c && grep -E 'lbu\s\$2,0\(\$4\)' /work/tests/little_endian.s

--- a/gcc-2.7.2.1.Dockerfile
+++ b/gcc-2.7.2.1.Dockerfile
@@ -30,7 +30,7 @@ RUN patch -u -p1 configure -i ../patches/configure.patch
 RUN patch -u -p1 config.sub -i ../patches/config.sub.patch
 RUN patch -u -p1 config/mips/mips.h -i ../patches/mipsel-2.7.patch
 
-RUN make -j cpp cc1 xgcc cc1plus g++ CFLAGS="-std=gnu89 -m32 -static"
+RUN make --jobs $(nproc) cpp cc1 xgcc cc1plus g++ CFLAGS="-std=gnu89 -m32 -static"
 
 COPY tests /work/tests
 RUN ./cc1 -quiet -O2 /work/tests/little_endian.c && grep -E 'lbu\s\$2,0\(\$4\)' /work/tests/little_endian.s

--- a/gcc-2.7.2.2.Dockerfile
+++ b/gcc-2.7.2.2.Dockerfile
@@ -30,7 +30,7 @@ RUN patch -u -p1 configure -i ../patches/configure.patch
 RUN patch -u -p1 config.sub -i ../patches/config.sub.patch
 RUN patch -u -p1 config/mips/mips.h -i ../patches/mipsel-2.7.patch
 
-RUN make -j cpp cc1 xgcc cc1plus g++ CFLAGS="-std=gnu89 -m32 -static"
+RUN make --jobs $(nproc) cpp cc1 xgcc cc1plus g++ CFLAGS="-std=gnu89 -m32 -static"
 
 COPY tests /work/tests
 RUN ./cc1 -quiet -O2 /work/tests/little_endian.c && grep -E 'lbu\s\$2,0\(\$4\)' /work/tests/little_endian.s

--- a/gcc-2.7.2.3.Dockerfile
+++ b/gcc-2.7.2.3.Dockerfile
@@ -28,7 +28,7 @@ RUN sed -i -- 's/include <varargs.h>/include <stdarg.h>/g' *.c
 RUN patch -u -p1 obstack.h -i ../patches/obstack-2.7.2.h.patch
 RUN patch -u -p1 config/mips/mips.h -i ../patches/mipsel-2.7.patch
 
-RUN make -j cpp cc1 xgcc cc1plus g++ CFLAGS="-std=gnu89 -m32 -static"
+RUN make --jobs $(nproc) cpp cc1 xgcc cc1plus g++ CFLAGS="-std=gnu89 -m32 -static"
 
 COPY tests /work/tests
 RUN ./cc1 -quiet -O2 /work/tests/little_endian.c && grep -E 'lbu\s\$2,0\(\$4\)' /work/tests/little_endian.s

--- a/gcc-2.7.2.Dockerfile
+++ b/gcc-2.7.2.Dockerfile
@@ -30,7 +30,7 @@ RUN patch -u -p1 configure -i ../patches/configure.patch
 RUN patch -u -p1 config.sub -i ../patches/config.sub.patch
 RUN patch -u -p1 config/mips/mips.h -i ../patches/mipsel-2.7.patch
 
-RUN make -j cpp cc1 xgcc cc1plus g++ CFLAGS="-std=gnu89 -m32 -static"
+RUN make --jobs $(nproc) cpp cc1 xgcc cc1plus g++ CFLAGS="-std=gnu89 -m32 -static"
 
 COPY tests /work/tests
 RUN ./cc1 -quiet -O2 /work/tests/little_endian.c && grep -E 'lbu\s\$2,0\(\$4\)' /work/tests/little_endian.s

--- a/gcc-2.8.0.Dockerfile
+++ b/gcc-2.8.0.Dockerfile
@@ -28,7 +28,7 @@ RUN patch -u -p1 obstack.h -i ../patches/obstack-2.8.0.h.patch
 
 RUN patch -u -p1 config/mips/mips.h -i ../patches/mipsel-2.8.patch
 
-RUN make cpp cc1 xgcc cc1plus g++ CFLAGS="-std=gnu89 -m32 -static"
+RUN make --jobs $(nproc) cpp cc1 xgcc cc1plus g++ CFLAGS="-std=gnu89 -m32 -static"
 
 COPY tests /work/tests
 RUN ./cc1 -quiet -O2 /work/tests/little_endian.c && grep -E 'lbu\s\$2,0\(\$4\)' /work/tests/little_endian.s

--- a/gcc-2.8.1.Dockerfile
+++ b/gcc-2.8.1.Dockerfile
@@ -28,7 +28,9 @@ RUN patch -u -p1 obstack.h -i ../patches/obstack-2.8.1.h.patch
 
 RUN patch -u -p1 config/mips/mips.h -i ../patches/mipsel-2.8.patch
 
-RUN make -j cpp cc1 xgcc cc1plus g++ CFLAGS="-std=gnu89 -m32 -static"
+RUN touch insn-config.h
+
+RUN make --jobs $(nproc) cpp cc1 xgcc cc1plus g++ CFLAGS="-std=gnu89 -m32 -static"
 
 COPY tests /work/tests
 RUN ./cc1 -quiet -O2 /work/tests/little_endian.c && grep -E 'lbu\s\$2,0\(\$4\)' /work/tests/little_endian.s

--- a/gcc-2.91.66.Dockerfile
+++ b/gcc-2.91.66.Dockerfile
@@ -33,7 +33,7 @@ RUN patch -u -p1 gcc/obstack.h -i ../patches/obstack-${VERSION}.h.patch
 RUN patch -u -p1 gcc/config/mips/mips.h -i ../patches/mipsel-2.8.patch
 
 RUN make -C libiberty/ CFLAGS="-std=gnu89 -m32 -static"
-RUN make -C gcc/ -j cpp cc1 xgcc cc1plus g++ CFLAGS="-std=gnu89 -m32 -static"
+RUN make -C gcc/ --jobs $(nproc) cpp cc1 xgcc cc1plus g++ CFLAGS="-std=gnu89 -m32 -static"
 
 COPY tests /work/tests
 RUN ./gcc/cc1 -quiet -O2 /work/tests/little_endian.c && grep -E 'lbu\s\$2,0\(\$4\)' /work/tests/little_endian.s

--- a/gcc-2.95.2.Dockerfile
+++ b/gcc-2.95.2.Dockerfile
@@ -33,7 +33,7 @@ RUN patch -u -p1 include/obstack.h -i ../patches/obstack-${VERSION}.h.patch
 RUN patch -u -p1 gcc/config/mips/mips.h -i ../patches/mipsel-2.8.patch
 
 RUN make -C libiberty/ CFLAGS="-std=gnu89 -m32 -static"
-RUN make -C gcc/ -j cpp cc1 xgcc cc1plus g++ CFLAGS="-std=gnu89 -m32 -static"
+RUN make -C gcc/ --jobs $(nproc) cpp cc1 xgcc cc1plus g++ CFLAGS="-std=gnu89 -m32 -static"
 
 COPY tests /work/tests
 RUN ./gcc/cc1 -mel -quiet -O2 /work/tests/little_endian.c && grep -E 'lbu\s\$2,0\(\$4\)' /work/tests/little_endian.s

